src/api/routes/discovery.js

```
@@ -47,13 +49,79 @@ discoveryRoutes.get("/chitty.json", async (c) => {
+        }
+      }
+      if (!sub) sub = name;
+
+      return {
+        name,
+        url,
+        sub,
+        health_url: service?.health_url,
+        status: service?.status,
+      };
+    }
```

ESLint

[error] Move function declaration to function body root.

(no-inner-declarations)